numba/tests/test_bytes.py

```diff
@@ -0,0 +1,41 @@
+from numba.tests.support import (TestCase, no_pyobj_flags, MemoryLeakMixin)
+from numba import njit
+import unittest
+
+
+def addbytes_len_usecase(a, b):
+    output = len(b''.join([a, b]))
+    return output
+
+
+BYTES_EXAMPLES = [
+    b"abcdef",
+    b"12345",
+    b"AAAA",
+    b"zzzzz",
+    b'z',
+    b'1',
+    bytes(5),
+    bytes(10)
+]
+
+
+class BaseTest(MemoryLeakMixin, TestCase):
+    def setUp(self):
+        super(BaseTest, self).setUp()
+
+
+class TestBytesData(BaseTest):
+    def test_addbytes_len(self, flags=no_pyobj_flags):
+        pyfunc = addbytes_len_usecase
+        cfunc = njit(pyfunc)
```

Comment:
Unless ` addbytes_len_usecase` is going to be reused at some future date, suggest defining it locally as an `njit` function and then using the `py_func` attribute to access the wrapped python function for use in testing.